test/e2e/functional/tests/e2e.go

```diff
@@ -27,19 +27,30 @@ import (
 	goclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var autoAssign = false
-var UseMetallbResourcesFromFile = false
-
-var OperatorNameSpace = consts.DefaultOperatorNameSpace
+var (
+	autoAssign                  = false
+	UseMetallbResourcesFromFile = false
+	OperatorNameSpace           = consts.DefaultOperatorNameSpace
+	controllerSelector          = "app.kubernetes.io/component=controller"
+	speakerSelector             = "app.kubernetes.io/component=speaker"
```

Comment:
can u add Label to the above controller and speaker to match with controller naming

Reply:
yep